content/cybersecurity/concepts/cryptography/cryptography.md

```diff
@@ -0,0 +1,28 @@
+# Cryptography 
```

Comment:
We're missing the metadata at the head of the entry. You can look at the source of other Cybersecurity entries ([like this one](https://github.com/Codecademy/docs/blob/main/content/cybersecurity/concepts/cyber-attack/cyber-attack.md)) to see what we need.

Also, remove the first h1 header. The Metadata title will provide the header for the entry.

Reply:
@SSwiniarski completed